ros_ethercat_model/include/ros_ethercat_model/ros_ethercat.hpp

```diff
@@ -44,6 +44,7 @@
 #include <sys/stat.h>
 #include <boost/algorithm/string/split.hpp>
 #include <boost/algorithm/string.hpp>
+#include <boost/thread/thread.hpp>
```

Comment:
No. In fact this is something that already was happening in the [main.cpp](https://github.com/shadow-robot/ros_ethercat/blob/indigo-devel/ros_ethercat_loop/src/main.cpp#L199) in the ros_ethercat_loop, but I removed it from the ros_control_robot loop.

So what I'm doing here is to add it again in RosEthercat (but only in the init function, that is called when loading the RosEthercat as a plugin.

This will be a low priority thread, as it is spawned before the control loop thread is given high priority (and by default they inherit the priority).